grumphp.yml

```diff
@@ -1,4 +1,6 @@
 grumphp:
+    git_hook_variables:
+        EXEC_GRUMPHP_COMMAND: 'docker run -e "TERM=xterm-256color"  --tty=true --rm -v $PWD:/code:cached pheature/pheature-php-dev:latest'
```

Comment:
Hey @xserrat I added a commit with a colored output example

Reply:
Oh!! Perfect 👏🏽  🚀

Reply:
I've just added also to the rest of the packages!! Thanks @kpicaza!!